pandas/io/tests/test_parsers.py

```diff
@@ -3,42 +3,151 @@
 
 # flake8: noqa
 
-from datetime import datetime
 import csv
 import os
-import sys
-import re
-import nose
 import platform
-
+import re
+import sys
+from datetime import datetime
 from multiprocessing.pool import ThreadPool
 
-from numpy import nan
+import nose
 import numpy as np
-from pandas.io.common import DtypeWarning
+import pandas.lib as lib
+import pandas.parser
+from numpy import nan
+from numpy.testing.decorators import slow
+from pandas.lib import Timestamp
 
-from pandas import DataFrame, Series, Index, MultiIndex, DatetimeIndex
+import pandas as pd
+import pandas.io.parsers as parsers
+import pandas.tseries.tools as tools
+import pandas.util.testing as tm
+from pandas import DataFrame, Series, Index, MultiIndex, DatetimeIndex, compat
 from pandas.compat import(
-    StringIO, BytesIO, PY3, range, long, lrange, lmap, u
+    StringIO, BytesIO, PY3, range, long, lrange, lmap, u, parse_date
 )
-from pandas.io.common import URLError
-import pandas.io.parsers as parsers
+from pandas.io.common import DtypeWarning, URLError
 from pandas.io.parsers import (read_csv, read_table, read_fwf,
                                TextFileReader, TextParser)
+from pandas.tseries.index import date_range
 
-import pandas.util.testing as tm
-import pandas as pd
 
-from pandas.compat import parse_date
-import pandas.lib as lib
-from pandas import compat
-from pandas.lib import Timestamp
-from pandas.tseries.index import date_range
-import pandas.tseries.tools as tools
+class Compression(object):
```

Comment:
Does this need to be called `CompressionTest` to get picked up? I know different test frameworks have different requirements.